README.md

```diff
@@ -12,11 +12,15 @@ There is a public calendar if you would like to subscribe to these events:
 
 ## Next meeting
 
-- 27 February 2025: ([Meeting Agenda](2025-02-27/agenda.md)) on the [MDN Discord](https://developer.mozilla.org/discord)
-- Content Meeting: 3rd March 2025
+- 26 March 2025: ([Meeting Agenda](2025-03-26/agenda.md)) on the [MDN Discord](https://developer.mozilla.org/discord)
+- Content Meeting: 17th March 2025
```

Comment:
Just a nit - we can stick to the cardinal format for consistency:
```suggestion
- Content Meeting: 17 March 2025
```

Reply:
Oh, no. I missed this! I'll fix this while adding agenda items. 👍